vault/README.md

```diff
@@ -36,11 +31,27 @@ helm install vault oci://ghcr.io/bank-vaults/helm-charts/vault
 To install the chart backed with a Consul cluster, use the following:
 
 ```bash
-helm install vault oci://ghcr.io/bank-vaults/helm-charts/vault --set vault.config.storage.consul.address="myconsul-svc-name:8500",vault.config.storage.consul.path="vault"
+helm install vault oci://ghcr.io/bank-vaults/helm-charts/vault \
+--set vault.config.storage.consul.path="vault"
```

Comment:
`\` is missing from the end of this line

Reply:
nice catch!